src/utils/calendar.js

```diff
@@ -79,3 +79,29 @@ export const parseInputToDate = (text, locale) => {
   if (localeDate.isValid()) return localeDate.format('YYYY-MM-DD');
   return '';
 };
+
+export const isDateInRange = (item, min, max) => {
+  const itemDate = moment(item);
+
+  if (min && max) {
+    return itemDate.isBetween(min, max, null, []);
+  }
+  if (min && !max) {
+    return itemDate.isSameOrAfter(min);
+  }
+  if (!min && max) {
+    return itemDate.isSameOrBefore(max);
+  }
+  return true;
+};
+
+export const getExampleDateStrings = () => ({
+  currentDate: moment().format('YYYY-MM-DD'),
+  preselectedDate: moment()
+    .add(3, 'days')
+    .format('YYYY-MM-DD'),
+  minDate: moment().format('YYYY-MM-DD'),
+  maxDate: moment()
+    .add(1, 'year')
+    .format('YYYY-MM-DD'),
+});
```

Comment:
Please add tests for this? Would make sense, right?

Reply:
Yeah. There are no existing tests for these calendar utils, but I'll start by adding for this one in this PR.

Reply:
Dun. The values returned from `getExampleDateStrings` are opinionated, so I didn't assert all their values, but only if they "made sense" for the examples.